vsts-gn.yml

```diff
@@ -26,13 +25,15 @@ phases:
       # BUILD_SOURCEVERSION commit, and so the checkout will fail. Find a
       # better solution for checking out the commit to be built.
       (cd src/electron; git fetch origin +"${BUILD_SOURCEBRANCH}"; git checkout "${BUILD_SOURCEVERSION}")
-      gclient sync --with_branch_heads --with_tags
+      gclient sync -v --with_branch_heads --with_tags
     name: gclient_sync
 
   - bash: |
       cd src
       export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
-      export SCCACHE_WRAPPER="`pwd`/libchromiumcontent/tools/sccache/aad2120/mac/sccache"
+      export SCCACHE_WRAPPER="`pwd`/electron/external_binaries/sccache"
+      export SCCACHE_BUCKET="electronjs-sccache"
+      export SCCACHE_TWO_TIER=true
```

Comment:
This should be removed

Reply:
Removed `SCCACHE_BUCKET` and `SCCACHE_TWO_TIER` as consistent with the discussion above.

Since `SCCACHE_WRAPPER` is `pwd`-dependent, I've left it in the latest push. Let me know if you want me to remove it too.